go.mod

```diff
@@ -34,19 +34,31 @@ require (
 )
 
 require (
+	github.com/NYTimes/gziphandler v1.1.1 // indirect
```

Comment:
Checking why d0569891 added these indirect references, it's via `github.com/openshift/library-go/pkg/controller/controllercmd` (our library-go dependency is old, but the `pkg/controller/controllercmd` dependency is new with this pull). 

```console
$ git --no-pager log --oneline -1
faf8310f (HEAD, origin/pr/1091) USC: Start insight receiver as PostStartHook
$ go version
go version go1.23.1 linux/amd64
$ go mod why github.com/NYTimes/gziphandler
# github.com/NYTimes/gziphandler
github.com/openshift/cluster-version-operator/cmd
github.com/openshift/library-go/pkg/controller/controllercmd
k8s.io/apiserver/pkg/server
k8s.io/kube-openapi/pkg/handler
github.com/NYTimes/gziphandler
```